tests/IntegratedTest.php

```diff
@@ -3,38 +3,81 @@
 namespace JeroenG\Packager\Tests;
 
 use Illuminate\Support\Facades\Artisan;
+use Symfony\Component\Process\Process;
 
 class IntegratedTest extends TestCase
 {
     public function test_new_package_is_created()
     {
+        // Also test downloading package-skeleton (the other test will use a cached copy)
+        config()->set('packager.cache_skeleton', false);
         Artisan::call('packager:new', ['vendor' => 'MyVendor', 'name' => 'MyPackage']);
-
         $this->seeInConsoleOutput('Package created successfully!');
+        $this->assertComposerPackageInstalled('MyVendor/MyPackage');
+        // Save the generated package for use in later tests
+        $this->storePackageAsFake();
     }
 
     public function test_get_existing_package()
     {
-        Artisan::call('packager:get',
-            ['url' => 'https://github.com/Jeroen-G/packager-skeleton', 'vendor' => 'MyVendor', 'name' => 'MyPackage']);
-
+        $this->assertComposerPackageNotInstalled('MyVendor/MyPackage');
+        Artisan::call('packager:get', [
+            'url' => 'https://github.com/Jeroen-G/packager-skeleton',
+            'vendor' => 'MyVendor',
+            'name' => 'MyPackage'
+        ]);
         $this->seeInConsoleOutput('Package downloaded successfully!');
+        $this->assertComposerPackageInstalled('MyVendor/MyPackage');
     }
 
+    /**
+     * @depends test_new_package_is_created
```

Comment:
I didn't know about `@depends`, looks very clever.

Reply:
I don't normally like writing tests that rely on other tests, but sometimes it's super convenient.